<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetGraphicsView</class>
 <widget class="QWidget" name="WidgetGraphicsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="layout">
     <item>
      <widget class="QCheckBox" name="optionsVisible">
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="hiddenWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="autoResize">
          <property name="text">
           <string>Auto resize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resize">
          <property name="text">
           <string>Resize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportAsSvg">
          <property name="text">
           <string>Export as SVG...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
